app/models/cloud_network.rb

```diff
@@ -7,4 +8,7 @@ class CloudNetwork < ActiveRecord::Base
   has_many   :cloud_subnets, :dependent => :destroy
   has_many   :security_groups
   has_many   :vms
+
+  # Use for virtual columns, mainly for modeling array and hash types, we get from the API
+  serialize :extra_attributes
```

Comment:
Hrm, `extra_attributes` seems like a bit like a cop-out catch all...

What data can we expect to store in `extra_attributes`?